src/Telegram/Types/Message/Message.php

```diff
@@ -216,6 +217,12 @@ class Message extends BaseType
      */
     public ?Sticker $sticker = null;
 
+    /**
+     * Optional.
+     * Message is a forwarded story
+     */
+    public ?Story $story = null;
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏Missing MessageType enum Mapping, and new update of the getType method.

I would also create a new listener like `onStory` or something like that, altho is also true that for now the object does not contain any useful info.